lxd/api_cluster.go

```diff
@@ -329,6 +447,117 @@ func clusterPutDisable(d *Daemon) Response {
 	return EmptySyncResponse
 }
 
+// Initialize storage pools and networks on this node.
+//
+// We pass to LXD client instances, one connected to ourselves (the joining
+// node) and one connected to the target cluster node to join.
+func clusterInitMember(d, client lxd.ContainerServer, memberConfig []api.ClusterMemberConfigKey) error {
+	data := initDataNode{}
+
+	// Fetch all pools current defined in the cluster.
+	pools, err := client.GetStoragePools()
+	if err != nil {
+		return errors.Wrap(err, "failed to fetch information about cluster storage pools")
```

Comment:
Errors must start with a capital (and never end with a period)

Reply:
Fixed.